superset/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM apache/superset
```

Comment:
It would be safer to hardcode the version we want to use, otherwise they may release a new version and break the deployment.